data/tabular/nomad_structure/meta.yaml

```
@@ -0,0 +1,126 @@
+      Assistant: The {pointgroup__names__noun} of the {#material|compound|solid!} is {pointgroup#}.
+    - |-
+      Question: {#What is the|What's the!} {density__names__noun} of the {#material|compound|solid!} with the {#CIF|CIF file|CIF card!} {cif#}?
+      Answer: The {density__names__noun} of the {#material|compound|solid!} with the {#CIF|CIF file|CIF card!} {cif#} is {density#} {density__units}.
+    - |-
+      User: I want to design a material with a particular {density__names__noun}, {spacegroup__names__noun}, and {#chemical formula|composition|reduced formula!}.
+      Assistant: {#Cool, |Awesome, |Great, |That sounds interesting, |!}I would need to know the {density__names__noun}, {spacegroup__names__noun}, and {#chemical formula|composition|reduced formula!} of the material you want to design.
+      User: The {density__names__noun} should be {density#} {density__units}, the {spacegroup__names__noun} should be {spacegroup#}, and the {#chemical formula|composition|reduced formula!} should be {formula#}.
+      Assistant: I {#recommend|suggest|propose|advise|!} the {#material|compound|solid!} with the {#CIF|CIF file|CIF card!} {cif#}.
+    - |-
+      Question: What is the {density__names__noun} of the {#material|compound|solid!} with the {#CIF|CIF file|CIF card!} {cif#}?
+      Answer: {density#} {density__units}.
+    - |-
+      Question: What is the {spacegroup__names__noun} of the {#material|compound|solid!} with the {#CIF|CIF file|CIF card!} {cif#}?
+      Answer: {spacegroup#}.
+    - |-
+      Question: What is the {#chemical formula|composition|reduced formula!} of the {#material|compound|solid!} with the {#CIF|CIF file|CIF card!} {cif#}?
+      Answer: {formula#}.
+    - |-
+      Question: What is the {spacegroup_number__names__noun} of the {#material|compound|solid!} with the {#CIF|CIF file|CIF card!} {cif#}?
+      Answer: {spacegroup_number#}.
+    - |-
+      User: The {#CIF|CIF file|CIF card!} of the {#material|compound|solid!} is {cif#}. {#Tell|show|Give|!} me the {#chemical formula|composition|reduced formula!}.
+      Assistant: "{#Certainly|Sure|Ofcourse!}, the {formula__names__noun} is {formula#}
+    - |-
+      User: The {#CIF|CIF file|CIF card!} of the {#material|compound|solid!} is {cif#}. {#Tell|show|Give|!} me the {density__names__noun}.
+      Assistant: "{#Certainly|Sure|Ofcourse!}, {density__names__noun} of the {#material|compound|solid!} is {density#} {density__units}
+    - |-
+      User: I want to design a {#material|compound|solid!} with a {density__names__noun} of {density#} {density__units}, and a {#chemical formula|composition|reduced formula!} of {formula#}.
+      Assistant: {#Cool, |Awesome, |Great, |That sounds interesting, |!}I suggest the {#material|compound|solid!} with the {#CIF|CIF file|CIF card!} {cif#}. {#Is there anything else I can do for you?|Do you need anything else?|Anything else?|!}
```

```suggestion
      Assistant: {#Cool, |Awesome, |Great, |That sounds interesting, |!}I suggest the {#material|compound|solid!} with the {#CIF|CIF file|CIF card!} {cif#}. {#Is there anything else I can do for you?|Do you need anything else?|Anything else?!}
```

will be strange if you then continue with `Indeed`